space/internal/components/aclnotifications/notifications.go

```diff
@@ -121,6 +144,14 @@ func (n *aclNotificationSender) sendNotification(ctx context.Context, aclNotific
 }
 
 func (n *aclNotificationSender) processRecords() {
+	ticker := time.NewTicker(time.Minute)
```

Comment:
In which cases we can have some records after 1 minute but not fully processed the tree? In case of local notifications?